src/main/resources/db/postgres/defaultDS/V1.0_017__pofim_169_foresporsel_endringer.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE FORESPOERSEL
+ADD FORESPOERSEL_TYPE VARCHAR(50);
```

Comment:
Når du legger til kolonne med not null så kan du bruke DEFAULT  (+ tilsvarende i entitet = ....)
ALTER TABLE FORESPOERSEL
ADD FORESPOERSEL_TYPE VARCHAR(50)
NOT NULL DEFAULT 'BESTILT_AV_FAGSYSTEM';

Da slipper du en operasjon update + set not null som ikke er så dyr her, men kan være vanskelig i store tabeller.